projects/aca-content/src/lib/components/search/search-results/search-results.component.scss

```diff
@@ -24,6 +25,7 @@ $contrast-gray: #646569;
     padding: 0 25px;
     flex-direction: row;
     align-items: center;
+    border-top: 1px solid #eee;
```

Comment:
I feel like maybe we should have a variable here for the border color, instead of a set hex code? I do see a variable already defined for border colors - --theme-border-color, although it needs to be confirmed whether it can be applied here or not.

Reply:
good point, thanks